electrumx/lib/coins.py

```diff
@@ -874,6 +874,24 @@ class BitcoinNolnet(BitcoinCash):
     RPC_PORT = 28332
     PEER_DEFAULT_PORTS = {'t': '52001', 's': '52002'}
 
+class Sumcoin(Coin):
```

Comment:
E302 expected 2 blank lines, found 1

Reply:
Changes have been made but still fails to build.  Suggestion please